optimize/qa/schema-integrity-tests/pom.xml

```diff
@@ -11,17 +11,33 @@
   <artifactId>optimize-schema-integrity-tests</artifactId>
 
   <properties>
-    <old.elasticsearch.port>9250</old.elasticsearch.port>
-    <new.elasticsearch.port>9200</new.elasticsearch.port>
+    <old.database.port>9250</old.database.port>
+    <new.database.port>9200</new.database.port>
   </properties>
 
   <dependencies>
     <dependency>
       <groupId>org.elasticsearch.client</groupId>
       <artifactId>elasticsearch-rest-high-level-client</artifactId>
-      <version>${elasticsearch.client.test.version}</version>
+      <version>7.17.24</version>
```

Comment:
I think we should have an issue for Ivan to remove this final usage of the RHLC (if we don't already)

Reply:
Created https://github.com/camunda/camunda/issues/22518